server/auth_function/test/conftest.py

```diff
@@ -96,46 +94,8 @@ def cognito_context():
     yield context
 
 
-@pytest.fixture(scope="session")
-def test_user_properties(cognito_event):
```

Comment:
Remove these fixture methods, as they rely on other fixture methods that is parametrized. Refactor them into the util file as regular function, so we can pass parameter in.